src/Middlewares/Middleware.php

```diff
@@ -111,7 +111,7 @@ function (ResourceContainer $container) {
                     $showCategoryFilter = false;
                 }
 
-                if ($isInSearchOrCategoryPage && !$this->searchService->aliveTest()) {
+                if ($isInSearchOrCategoryPage && !true) {
```

Comment:
Guess this was changed for debugging?

Seems to be for the whole file

Reply:
Since the alive test is now being done by the actual findologic-api - not sure what to do here ( maybe the api has a separate method to check if the API is alive?)